tools/configuration_helpers

```diff
@@ -66,6 +66,11 @@ set_local_configuration() {
     # DEFAULT_MAIN_MISSION_TIME: Allows for short missions for testing. 
     export DEFAULT_MAIN_MISSION_TIME=${DEFAULT_MAIN_MISSION_TIME:-600} 
 
+    # DIFFICULTY: Level of difficulty for mission. 3 for easy.
+    #                                              5 for medium.
+    #                                              7 for hard.
+    export DIFFICULTY=${DIFFICULTY:-5}
```

Comment:
Since we are exporting this variable here, we need to add an argument to the invocation of `runExperiment` (lines 241-243 of `tools/run_session` like `--level_of_difficulty=${DIFFICULTY}`? That will make `runExperiment` respect the difficulty level we set with the environment variable.

Reply:
Fixed.

I added to line 74 tools/configuration_helpers:
runExperiment_args="--level_of_difficulty=${DIFFICULTY}"

I had originally put these commands in run_session, line 246 (see Slack conversation, but it was in the loop of  "if [[ $CURRENT_MISSION -gt 0 ]]; then..." That was what I had commented out, then later deleted.